int-set/src/lib.rs

```diff
@@ -0,0 +1,165 @@
+mod bitpage;
```

Comment:
we should have some `//!` top level comment here, this is what would show up on the main page when rendering the docs.